src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -402,6 +416,10 @@ export default {
 				return null
 			}
 
+			if (!this.isScreensharingAllowed) {
+				return t('spreed', 'No screensharing')
+			}
```

Comment:
We should simply hide the screnshare button

Reply:
OK. Then I guess that audio and video buttons should be fully hidden too when publishing is not allowed, right?

Reply:
I would keep those to signal that they are not allowed to enable it. But having 3 buttons telling you that feels unnecessary.
Or do we want to split screenshare permissions from others. Then it might make sense

Reply:
So what's the plan here? We just keep all buttons?

Reply:
An alternative to hide them suggested by @ma12-co would be to keep them, but change the `No audio\camera\screensharing` tooltip to `You are not allowed to send audio/video/share your screen`. That way the users could know why they can not enable the microphone/camera/share the screen, as if there are no buttons they could think that the UI is broken.

Personally I prefer that approach to hiding the buttons, but I would be fine hiding them too. I have no strong preference.